nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -189,9 +189,14 @@ let
           mkdir /boot/grub
           echo '(hd0) /dev/vda' > /boot/grub/device.map
 
-          # Install GRUB and generate the GRUB boot menu.
+          # Install bootloader
```

Comment:
So this is not grub-specific anymore? I still see grub-specifics above…

Reply:
Rearranged comments to make this more clear.

Reply:
Thanks, this makes much more sense now!